group_project/group_project.py

```diff
@@ -136,15 +137,30 @@ def milestone_dates(self):
         group_activity = GroupActivity.import_xml_string(self.data, self.is_admin_grader)
         return group_activity.milestone_dates
 
+    def _get_project_api_implementation(self):
+        location = getattr(settings, 'GROUP_PROJECT_API_LOCATION', self._default_project_api_location)
+
+        split = location.split('.')
+        project_api_module, project_api_classname = ".".join(split[:-1]), split[-1]
```

Comment:
Nit: `project_api_module, project_api_classname = location.rsplit('.', 1)` would be a simpler replacement for these two lines